src/store/slices/classifierSlice.ts

```diff
@@ -56,6 +64,12 @@ export const classifierSlice = createSlice({
     predict(state, action: PayloadAction<{}>) {
       state.predicting = true;
     },
+    evaluate(
+      state,
+      action: PayloadAction<{
+        onOpenEvaluateDialog: () => void;
```

Comment:
I think this should probably be renamed. The callback passed here is the thing that *causes* `open` to be set to `true` in `EvaluateClassifierListItem`. This name however implies it's a function you call when `open` has *already* been set to `true`

Reply:
how about `setOpenEvaluateDialog`?

Reply:
👍